change/@fluentui-react-rating-preview-098152fc-985c-46de-aee3-2e1e7917e969.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30520&commitId=c479c199b690f48ac298d1c67f996fe93bc89ca0&env=prod&clientType=FLUENTUI)


  <details><summary><b>RatingDisplay Converged</b> 17 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|240381| Changed|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|238472| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|238410| Changed|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|237735| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|240914| Changed|
|RatingDisplay Converged.Marigold with half value.rest.chromium.png|240428| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|237762| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|240379| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|242081| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|240232| Changed|
|RatingDisplay Converged.Compact.rest.chromium.png|240502| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|237814| Changed|
|RatingDisplay Converged.no value.rest.chromium.png|240526| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|240313| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|238472| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|240343| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|240340| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>